### ISSUE TITLE
The discriminator loss for generator is always one

### ISSUE THREAD

**Issue by csxmli** [phillipi/pix2pix#57]
I rewrite this unet_128 to caffe prototxt and all the parameter is as soon as possible similar to this code. Also with the discriminator.
When I try to train this caffe net with one generator and one discriminator in training phase and the batch size is 1, I found that the discriminator loss is very small and accuracy is 1. I check my code carefully and found nothing wrong.
So my question is, is there any differenct between torch and caffe? I know batch norm layer has something different, are there any other place to notice?
Thank you for your reply.
